release/release.go

```diff
@@ -168,6 +171,36 @@ func getWSLVersion() int {
 	return 2
 }
 
+type snapYaml struct {
```

Comment:
all the code using here needs some XXX saying that we hope to have a more stable implementation of this as checking for exactly core22-desktop is fragile going forward